pages/core-api/transaction-service-overview.mdx

```diff
@@ -16,3 +16,8 @@ Additionally, the Safe\{Core\} SDK provides the [API Kit](../sdk/api-kit), a Typ
 ## Getting started
 
 Are you new to our API and not sure where to get started? We recommend heading over to the [guides](../core-api/transaction-service-guides/transactions.mdx) on the Safe Transaction Service API.
+
+## Resources
+
+- Learn about the [tech stack and how to run the service](./api-safe-transaction-service.mdx).
+- Check the [GitHub repository](https://github.com/safe-global/safe-transaction-service) (Python).
```

Comment:
These resources are only related with the Transaction Service but it is confusing as it is not specified.